src/CalculatorUITestFramework/HistoryPanel.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorUITestFramework
+{
+    public class HistoryPanel : ApplicationBase
+    {
+        public WindowsElement HistoryLabel => TryFindElementByAccessibilityId("HistoryLabel");
+        public WindowsElement HistoryListView => TryFindElementByAccessibilityId("HistoryListView");
+        public WindowsElement ClearHistoryButton => TryFindElementByAccessibilityId("ClearHistory");
+
```

Comment:
Do you think people will need to access these directly in test code?  Might be worth making private instead.

Reply:
The could need to. It depends on what they are trying to test in the History panel.